sandy-city-utah/targetsolutions-user-sync
# feat(validate): validate users and prepare update

### What Changed?
  - add more validation information
  - prepare more fully the data for the sync
  - add better logging throughout the application
  - fix a few issues not previously found

### Issues Addressed:
  - P21312-17

### Issue Testing:
  - Manual Developer Testing

### Test Results:
1. run process
1. See the CSV export file created
1. See the logs kept
